### PR TITLE
Fixed hello world templates

### LINKED ISSUE #16: Fix Hello World template for a few languages
A few languages have syntax errors in the hello world template. Should be a simple fix. 

Currently, the languages that need fixing:
- [ ] C# 
- [ ] Haskell
- [ ] Java
- [ ] Lisp
- [ ] Objective-C 
- [ ] PHP

### PR DESCRIPTION
**Pull request by mikeesto**
Resolves #16 
Also fixed the template for C on first load.

There does seem to be an issue on the backend when compiling for Lisp. I think it would be best if that is handled in a separate PR as it does not involve the templates. A new issue will need to be created. 

Thanks :)

### CODE REVIEW

**Comment by kaustubhhiware** [comment]
Great work @MichaelE1 :smile: 

Did you check if the templates are working on the IDE ?
I'm running into an error with C#
image


Please check for all templates

**Review by kaustubhhiware — CHANGES_REQUESTED** [review]
Great work @MichaelE1 :smile: 

Did you check if the templates are working on the IDE ?
I'm running into an error with C#, the herokuapp also has incorrect template for it
image


Please check for all templates

**Comment by mikeesto** [comment]
Hi @kaustubhhiware . Yes, I did check it on the IDE. It looks like your browser may be loading a cached version as that template has been changed in my commit (see lines 36-48).

Please either delete the cached files or view the IDE in incognito mode in Chrome (or equivalent for another browser). Thanks!

**Comment by kaustubhhiware** [comment]
Sorry, my bad @MichaelE1 :sweat_smile: 
The templates are correct
However I was thinking, we should have basic instructions for reading line as in java and php. What's your opinion ?
Also pinging @kingofools @Demfier for opinions

**Comment by kaustubhhiware** [comment]
@Demfier do you think the commented lines for input should be there?

**Comment by Demfier** [comment]
@kaustubhhiware - Yeah, that would be great. Someone new to the language gets instant info!
Apart from this, can we also frame the templates such that they use user defined functions. Something like this:

```python
def hello(input="Hello world")
    return(input)

input_string = raw_input("Enter input line:\n")
print(hello(input_string))
```

Now this includes:
1) how to define a function
2) how to take command-line input
3) how to call a function
4) how to output

in python.

Let me know your thoughts on this @MichaelE1 @kaustubhhiware @kingofools

**Comment by kaustubhhiware** [comment]
@MichaelE1 please add back the commented out lines in Java, php; they are required for convenience.
It would help greatly if @Demfier 's suggestions can be put into use.

**Comment by mikeesto** [comment]
Thanks @Demfier and @kaustubhhiware for your feedback. I agree with @Demfier 's suggested format. In particular I think it would be good to have consistency across all of the language templates.

However, I do think that this is now beyond the scope of issue #16 that this PR seeks to resolve. This PR fixes the current syntax issues for the languages that were listed in #16 . Updating the templates for all of the languages to the new format would require additional work that I'm unfortunately not able to currently provide.

I think it would be best if either a) I add the comments back in as suggested by @kaustubhhiware and then this PR is accepted as fixing the syntax errors, or b) this PR is closed and issue #16 is updated to include the need to add the new format for all of the languages. 

I'm happy to be guided by your decision :+1:

**Comment by kaustubhhiware** [comment]
@MichaelE1 what you point out is indeed right.
I think it would be beyond the scope of this PR or issue #16 to add some of these changes.

Just adding back the removed commented code should suffice for this P.R.
Another issue can be made from the suggestions @Demfier  made.

**Comment by mikeesto** [comment]
@kaustubhhiware I've added back the removed comments for Java and PHP. Ready for testing :)

**Comment by kaustubhhiware** [comment]
@MichaelE1 need to restore comments for C# as well.

**Comment by mikeesto** [comment]
@kaustubhhiware Restored as requested :+1:

File: templates/static/js/terminal_fill.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ function terminal(el)
             return 0;\n\
         }';
     } else if (value == 2) {
-        textarea.value = 'using System;\n\
+        textarea.value = textarea.value = 'using System;\n\
```
Comment (kaustubhhiware) [inline templates/static/js/terminal_fill.js]: `textarea.value` is repeated here. Please rectify.

Comment (mikeesto) [reply templates/static/js/terminal_fill.js]: Sorry about that. Fixed and also fixed the spacing in a comment.

**Comment by kaustubhhiware** [comment]
Seems we're almost done here!
Just squash your commits into a single commit and I'll merge it ASAP.

**Comment by mikeesto** [comment]
@kaustubhhiware Squashed and ready to merge :100:

**Review by kaustubhhiware — APPROVED** [review]

**Comment by kaustubhhiware** [comment]
Seems good! Merging this in. Thanks for your work and patience, @MichaelE1 😄

**Pull request merged**